tests/cli/test_env.py

```diff
@@ -375,12 +375,11 @@ def test_update_env_no_action_json_output(
 
 
 @pytest.mark.integration
-def test_remove_dry_run(env1: str, conda_cli: CondaCLIFixture):
+def test_remove_dry_run(tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture):
     # Test for GH-10231
-    create_env(ENVIRONMENT_CA_CERTIFICATES)
-    conda_cli("env", "create")
-    conda_cli("env", "remove", f"--name={env1}", "--dry-run")
-    assert env_is_created(env1)
+    with tmp_env() as prefix:
```

Comment:
This is no longer the same test since `ca-certificates` is not installed, was this an intentional change?